crypto/x509/x509_vfy.c

```diff
@@ -366,20 +367,20 @@ static int check_issued(ossl_unused X509_STORE_CTX *ctx, X509 *x, X509 *issuer)
  * Alternative get_issuer method: look up from a STACK_OF(X509) in other_ctx.
  * Returns -1 on internal error.
  */
-static int get_issuer_sk(X509 **issuer, X509_STORE_CTX *ctx, X509 *x)
+static int get_issuer_sk(X509 **p_res, X509_STORE_CTX *ctx, X509 *x)
```

Comment:
I'm not sure this is a better name for the parameter.

Reply:
I thought that the `p_` prefix may help to highlight that this is a double pointer,
and `res` makes clear that this is for the result.

Reply:
I'd really prefer keeping the original. IMO it is clear enough.